src/python/WMComponent/DBS3Buffer/DBSBufferBlock.py

```diff
@@ -232,10 +232,19 @@ def setProcessingVer(self, procVer):
         Set the block's processing version.
         """
         # compatibility statement for old style proc ver (still needed ?)
-        if procVer.count("-") == 1:
-            self.data["processing_era"]["processing_version"] = procVer.split("-v")[1]
+        pver = procVer
```

Comment:
Valentin, I would suggest to drop this compatibility statement. It's been many years and we definitely do not have any of those ancient workflows still active in the system.

Reply:
this I agree may go, I only preserved the original logic, will drop